### PR TITLE
add '@throws' PHPDoc annotation

### PR DESCRIPTION
**Pull request by muli**

### CODE REVIEW

File: lib/PhpZendClientGenerator.php [hunk 1]
```diff
@@ -580,6 +580,7 @@ function writeAction($serviceId, $serviceName, DOMElement $actionNode, $plugin =
 		$this->appendLine();
 		$this->appendLine("	/**");
 		$this->appendLine("	 * @return $returnType");
+		$this->appendLine("	 * @throws Kaltura_Client_Exception");
```
Comment (atarsh) [inline lib/PhpZendClientGenerator.php]: @muli what about Kaltura_Client_ClientException ?

Comment (muli) [reply lib/PhpZendClientGenerator.php]: Right. Added.

**Comment by muli** [comment]
Hey @tan-tan-kanarek,

Can this be merged? Are there any other actions to be done before/after merging?

Thanks!

**Comment by tan-tan-kanarek** [comment]
Please pull request against default branch, currently it's Naos-14.2.0.

**Comment by muli** [comment]
Done.

**Pull request merged**